azure-communication-ui/azure-communication-ui/src/calling/java/com/azure/android/communication/ui/calling/presentation/fragment/calling/participant/grid/ParticipantGridCellViewModel.kt

```diff
@@ -16,17 +17,21 @@ internal class ParticipantGridCellViewModel(
     isMuted: Boolean,
     isSpeaking: Boolean,
     modifiedTimestamp: Number,
+    participantStatus: ParticipantStatus?,
 ) {
+    private var isOnHoldStateFlow = MutableStateFlow(isOnHold(participantStatus))
```

Comment:
I get the feeling that we'd want to push participantStatus via the stateflow, and then just have isOnHold as a getter with the check against participant status.

It seems fairly viable that we may have further GridCell logic based on ParticipantStatus down the road. It'd make it easier to add isRinging or other possible states for the vm later.

Reply:
I think expose bare minimum that is directly used in the view. View should not get a complex object and then make decisions based on that.

Reply:
I think, just returning a specific state helps to reduce logic in view. Currently, view needs only hold state.